### ISSUE TITLE
default color doesn't work inside array 

### ISSUE THREAD

**Issue by grafst** [json-editor/json-editor#1257]
### General information

- json-editor version: latest (2.8.0)

### Expected behavior
shows the color defined in 'default' (analogue to a color field outside of an array)

### Actual behavior
shows black (empty)

### Steps to reproduce the behavior

Direct link to example: https://is.gd/9pQBqo

```javascript
{
    "title": "default color array demo",
    "type": "object",
    "properties": {
        "colors": {
            "type": "array",
            "format": "table",
            "title": "colors",
            "uniqueItems": true,
            "items": {
                "type": "object",
                "properties": {
                    "text": {
                        "type": "string",
                        "default": "X"
                    },
                    "color": {
                        "type": "string",
                        "format": "color",
                        "default": "#fefaac"
                    }
                }
            }
        },
        "color": {
            "type": "string",
            "format": "color",
            "default": "#fefaac"
        }
    }
}
```

**Comment by schmunk42** [comment]
This is only with format table, right?

List: https://is.gd/uX4eqU
Tabs: https://is.gd/oVPJnz

**Comment by grafst** [comment]
yes, looks like it is only with table

**Comment by schmunk42** [comment]
@germanbisurgi is this an easy fix?
@grafst or could you come up with a PR?

**Comment by germanbisurgi** [comment]
I'm on it. It's easy to fix but i was difficult to find.

**Comment by germanbisurgi** [comment]
Fixed in #1257

**Comment by grafst** [comment]
wow really cool it's already fixed!! 🎉 
Yes I looked trough the code but I didn't find out where the problem was...
now I love the json-editor even more 😍
